packages/mui-material/src/ButtonBase/ButtonBase.js

```diff
@@ -47,24 +47,28 @@ export const ButtonBaseRoot = styled('button', {
   borderRadius: 0,
   padding: 0, // Remove the padding in Firefox
   cursor: 'pointer',
-  userSelect: 'none',
   verticalAlign: 'middle',
   MozAppearance: 'none', // Reset
   WebkitAppearance: 'none', // Reset
   textDecoration: 'none',
   // So we take precedent over the style of a native <a /> element.
   color: 'inherit',
+  '&:active': {
```

Comment:
Doesn't work on Firefox macOS. 

But  this seems to do it.

```suggestion
  '&:focus': {
```

---

This deserves a comment, it looks strange without context.